src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -4895,6 +4895,41 @@ To record a specific window, enter the hotkey with the Alt key in the opposite m
   </data>
   <data name="SettingsPage_NewInfoBadge.Text" xml:space="preserve">
     <value>NEW</value>
-	<comment>Must be all caps</comment>
+    <comment>Must be all caps</comment>
+  </data>
+  <data name="Hosts_Backup_GroupSettings.Header" xml:space="preserve">
+    <value>Backup</value>
+  </data>
+  <data name="Hosts_Backup.Header" xml:space="preserve">
+    <value>Backup hosts file</value>
+    <comment>"Hosts" refers to the system hosts file, do not loc</comment>
+  </data>
+  <data name="Hosts_Backup.Description" xml:space="preserve">
+    <value>Automatically create a backup of the hosts file when you save for the first time in a session</value>
+    <comment>"Hosts" refers to the system hosts file, do not loc</comment>
+  </data>
+  <data name="Hosts_Backup_Location.Header" xml:space="preserve">
+    <value>Location</value>
+  </data>
+  <data name="Hosts_ButtonSelectLocation.Text" xml:space="preserve">
+    <value>Select location</value>
+  </data>
+  <data name="Hosts_Delete_Backup.Content" xml:space="preserve">
+    <value>Automatically delete old backups</value>
+  </data>
+  <data name="Hosts_Backup_DaysToKeep.Header" xml:space="preserve">
+    <value>Days to keep</value>
+  </data>
+  <data name="Hosts_Backup_CopiesToKeep.Header" xml:space="preserve">
+    <value>Backups to keep</value>
+  </data>
+  <data name="Hosts_Backup_CopiesToKeep.Description" xml:space="preserve">
+    <value>Specify a fixed number of backup copies to retain</value>
+  </data>
+  <data name="Hosts_Backup_DaysToKeep.Description" xml:space="preserve">
+    <value>Specify the number of days to retain backups</value>
+  </data>
+  <data name="Hosts_DeleteBackupWarning.Title" xml:space="preserve">
+    <value>Backups will never will be kept indefinitely with the current configuration</value>
```

Comment:
```suggestion
    <value>Backups will be kept indefinitely with the current configuration</value>
```

Reply:
Thanks for catching this (haven't reviewed the PR yet 😄).
What I wanted here is **Backups will be kept indefinitely with the current configuration**.

Reply:
The UI is confusing: If I keep my backups 0 days, then I have no backups. 🤔